gsa/src/web/graphql/notes.js

```diff
@@ -45,19 +45,25 @@ export const GET_NOTES = gql`
     ) {
       edges {
         node {
+          active
```

Comment:
If you change the query the corresponding graphql mocks have to be adjusted as well.